src/plugins/auto/tests/unit/stateful_model_test.cpp

```diff
@@ -249,19 +277,78 @@ const std::vector<StatefulModelConfigParams> testConfigs = {
                               std::map<std::string, bool>{{"GPU", true}},
                               false,
                               std::vector<std::pair<std::string, int>>{{"GPU", 1}}},
-
+    StatefulModelConfigParams{"GPU",
+                              true,
+                              true,
+                              std::map<std::string, bool>{{"GPU", true}},
+                              false,
+                              std::vector<std::pair<std::string, int>>{{"GPU", 1}}},
+    StatefulModelConfigParams{"GPU",
+                              true,
+                              true,
+                              std::map<std::string, bool>{{"GPU", true}},
+                              true,
+                              std::vector<std::pair<std::string, int>>{{"GPU", 1}}},
+    StatefulModelConfigParams{"GPU",
+                              true,
+                              true,
+                              std::map<std::string, bool>{{"GPU", false}},
+                              true,
+                              std::vector<std::pair<std::string, int>>{{"GPU", 1}}},
```

Comment:
does the test check that enable start up fall back is disabled when CPU and GPU both support stateful?

Reply:
Updated.